pennylane/ops/qubit/parametric_ops.py

```diff
@@ -0,0 +1,1170 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the discrete-variable quantum operations that are the
+core parameterized gates.
+"""
+import cmath
+import functools
+
+# pylint:disable=abstract-method,arguments-differ,protected-access
+import math
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import AnyWires, DiagonalOperation, Operation
+from pennylane.templates.decorator import template
+from pennylane.utils import expand, pauli_eigs
+from pennylane.wires import Wires
+
+from .non_parametric_ops import PauliX, PauliY, PauliZ, Hadamard, CNOT
```

Comment:
Could it be worth using an absolute import here? Linting seems to give `E0402: Attempted relative import beyond top-level package (relative-beyond-top-level)` here.

Reply:
This might be good to address still, just changing it to an absolute import. 🙂